ocaml-lsp-server/src/dune.ml

```diff
@@ -643,7 +643,10 @@ let poll active last_error =
       | `Skip -> Fiber.return ()
       | `Print ->
         let message =
-          sprintf "failed to poll dune registry. %s" (Printexc.to_string exn)
+          sprintf
+            "failed to poll dune registry. %s \n\
+             Maybe you are not running dune in watch mode?"
```

Comment:
Should the message come before the exception perhaps?

Reply:
That's how I had it at first, but "error description -> exception -> potential solution" seemed like a more natural order. Otherwise, the exception feels randomly stuck onto the end.